s4e-backend/src/main/java/pl/cyfronet/s4e/bean/ProductCategory.java

```diff
@@ -0,0 +1,29 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class ProductCategory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotEmpty
+    private String name;
+
+    @NotEmpty
+    private String label;
+
+    @NotNull
+    @Column(name="url")
```

Comment:
This annotation is unnecessary. Also, if you want to use the default value in builder you should annotate the field with `@Builder.Default`.